src/worldcereal/job.py

```diff
@@ -194,19 +201,28 @@ def generate_map(
     if job_options is not None:
         JOB_OPTIONS.update(job_options)
 
-    # Compile filename of final product
-    proc_level = "cleaned" if postprocess_parameters.enable else "raw"
-    filename = f"{product_type.value}-{proc_level}"
-
     # Execute the job
     job = classes.execute_batch(
         out_format=out_format,
         job_options=JOB_OPTIONS,
         title="WorldCereal [generate_map] job",
         description="Job that performs end-to-end WorldCereal inference",
-        filename_prefix=filename,
+        filename_prefix=product_type.value,
     )
 
+    # Get look-up tables
+    luts = {}
+    luts[WorldCerealProductType.CROPLAND.value] = load_model_lut(
+        cropland_parameters.classifier_parameters.classifier_url
+    )
+    if (
```

Comment:
there's no way to avoid if statements here I guess?

Reply:
not that I can think of no...